src/components/Form/FieldCurrency/FieldCurrency.spec.tsx

```diff
@@ -0,0 +1,145 @@
+import { expect, test, vi } from 'vitest';
+import { z } from 'zod';
+
+import { render, screen, setupUser } from '@/tests/utils';
+
+import { FormField } from '..';
+import { FormMocked } from '../form-test-utils';
```

Comment:
We want to avoid `@/` or the auto import did that?

Reply:
We don't want to link the import to the project so relative import is easier here I think.